Assets/Scripts/GameState/PauseManager.cs

```diff
@@ -9,6 +9,9 @@ public class PauseManager : MonoBehaviour
     [SerializeField]
     public GameObject ui;
 
+    // An array of all audiosources in the game.
+    public AudioSource[] audioSources;
```

Comment:
This should probably be made private 🤔
```suggestion
    [SerializeField]
    private AudioSource[] audioSources;
```